vsts/sdl.yaml

```diff
@@ -71,11 +71,11 @@ phases:
       codebase: NewOrUpdate
       tsaEnvironment: PROD
       codeBaseName: 'Azure-Iot-SDK-Java-Master'
-      notificationAlias: 'timtay@microsoft.com, prmathur@microsoft.com, jasminel@microsoft.com'
-      codeBaseAdmins: 'REDMOND\timtay;REDMOND\prmathur;REDMOND\jasminel'
+      notificationAlias: 'timtay@microsoft.com, jamdavi@microsoft.com, David.R.Williamson@microsoft.com, Basel.Rustum@microsoft.com, azabbasi@microsoft.com, Abhipsa.Misra@microsoft.com, vinagesh@microsoft.com'
+      codeBaseAdmins: 'REDMOND\timtay;REDMOND\jamdavi;REDMOND\azabbasi'
```

Comment:
I'm not sure how to do the alias for users without single "word" aliases. I'm also not sure if the codeBaseAdmins even matters since the notificationAlias is the mechanism that alerts us to issues

Reply:
I'm drwill@microsoft.com

Reply:
could you use aziotdndmgdsdk@microsoft.com to catch us all and be resilient to team composition changes?

Reply:
Good idea